docs/MongoDB.md

```diff
@@ -101,14 +101,31 @@ WAL-G mongodb extension currently supports these commands:
 
 ### ``backup-push``
 
-Creates new backup and send it to storage.
+Creates new logical backup and send it to storage.
 
 Runs `WALG_STREAM_CREATE_COMMAND` to create backup.
 
 ```bash
 wal-g backup-push
 ```
 
+### ``binary-backup-push``
+
+Creates new binary backup and send it to storage.
+
+This command uses [backup cursors functionality of WiredTiger](https://source.wiredtiger.com/develop/backup.html)
+to get list of files in local file system, that stay locked while backup cursor is opened.
+Applications may continue to read and write the databases while a binary snapshot is taken.
+
+Note: In this mvp version we support replicaset cluster only (non sharded).
+
+This functionality inspired by article 
```

Comment:
Hello!
Thanks for the functionality!

I think it's worth pointing out explicitly that this feature will not work for the community version of MongoDB.